docs/getting-started-kubernetes.rst

```diff
@@ -156,142 +156,256 @@ Below are deployment examples, which the ``values.yaml`` file provided to the He
 Network Operator Deployment with RDMA Shared Device Plugin
 ----------------------------------------------------------
 
-Network operator deployment with the default version of the OFED driver and a single RDMA resource mapped to ens1f0 netdev.:
+First install the Network Operator with NFD enabled:
 
-``values.yaml`` configuration file for such a deployment:
+``values.yaml``:
 
 .. code-block:: yaml
 
     nfd:
       enabled: true
-    sriovNetworkOperator:
-      enabled: false
-    # NicClusterPolicy CR values:
-    deployCR: true
-    ofedDriver:
-      deploy: true
 
-    rdmaSharedDevicePlugin:
-      deploy: true
-      resources:
-        - name: rdma_shared_device_a
-          ifNames: [ens1f0]
+Once the Network Operator is installed create a NicClusterPolicy with
+* DOCA driver
+* RDMA Shared device plugin configured to a netdev with name ens1f0.
+
 
-    sriovDevicePlugin:
-      deploy: false
+Note: You may need to change the interface names in the NicClusterPolicy to those used by your target nodes.
+
+.. parsed-literal::
```

Comment:
```suggestion
.. code-block:: yaml
```

shouldn't this be rather a `yaml` code block?

Reply:
Unfortunately no - as we need to substitute the version numbers here. That functionality doesn't work with code-block. So here we're trading off the variable substitution for syntax highlighting.

Reply:
https://github.com/Mellanox/network-operator-docs/issues/28

Reply:
Okay, wasn't aware of that. Thanks